migrations/README.md

```diff
@@ -64,12 +64,28 @@ database with a given group of work, write a migration.
   - When in doubt, attach CSVs or screenshots from data clips generated from
   before the migration and after it was complete to the ticket.
 - Prefer having another engineer verify the data change, [[Four Eyes Principle]]
-- Use the `DataChange` utility to track changes.
 - Use SQL, not `ActiveRecord` models, in data migrations.
-- Depending on need data migrations fall into three use cases as follows.
 
+### Tracking Changes
+
+In order to know what was changed and when we should track any migrations run
+in our applications.
+
+- Use the `DataChange` utility to track changes.
+- Scale the usage of `DataChange` based on the number of records being migrated
+  across all environments and clients.
+  - With smaller data changes
+    - `pre` and `post` queries should be summaries, not all records.
+    - `change` query include helpful `returning` data.
+  - With large data changes
+    - `pre` and `post` queries should be summaries, not all records.
+    - `change` query should `disable_streaming`.
```

Comment:
The problem here is that with batching, we end up with one DataChange record per batch loop, and that can be hundreds or thousands of DataChange reports. This is just clutter at that point. 

I think we may want to consider using DataChange only when we are running one query, and just not track data in very large batched change sets. For those cases we can encourage the use of a `lib/tasks/tmp` rake task that can be run before and after to show that the data changes were indeed successful.

WDYT?

Reply:
Okay, addressed. I think we should still keep the pre and post tasks of the `DataChange`, so I made that modification along with adding batching of the actual change.